CHANGELOG.md

```diff
@@ -35,11 +34,12 @@ Other breaking changes:
 * [CHANGE] Cleanup NFS metrics #834
 * [CHANGE] Only report core throttles per core, not per cpu #836
 * [CHANGE] Treat custom textfile metric timestamps as errors #769
-* [CHANGE] Drop exec_ in boot_timestamp_seconds on *bsd #839
+* [CHANGE] Drop `exec_` in `boot_timestamp_seconds` on BSD #839
```

Comment:
How does this work with the line about 901 below?

Reply:
It's two different refactorings.

Reply:
They're touching the same metric though, I'd collapse it down to the end state. As-is it's confusing.